### ISSUE TITLE
Template with errors and warnings when uploading it to overleaf without modifications.

### ISSUE THREAD

**Issue by PedroOliveiraFernandes** [matpato/thesisisel#3]
![image](https://user-images.githubusercontent.com/17546509/194108219-b1bf3328-d221-4478-8941-6d3e1bc308d0.png)
So the image above was an upload of the master branch straight to overleaf without modifications. These errors and warnings happens when using the vscode and intelij plugins too.
Are there any steps missing that could be added to the readme file in order to build the project without problems?

Kind regards

**Comment by pedroamarocosta** [comment]
Found the exact same warnings (no errors, though) on the Overleaf public version. 32 warnings cannot be a good sign to start your work. Highly recommend to stay away before a major refactoring is done.

**Comment by daniel-m-carvalho** [comment]
Yes. I also have the same problem. I tried to use the iselthesisv4.0.2.zip directly on overleaf and it appears errors and warnings as you can see in the image from the previous comment.

**Comment by matpato** [comment]
Dear Daniel, You have to use the last version and not the iselthesisv4.0.2.zip. Could you explain better the errors you have. Because I downloaded in last minutes, put them in my overleaf account and everything works fine. Regarding to the warnings it’s normal, because some package have been installed and not are used, you can comment it. 
However, in your case you must use the new version and not the last one because of the new rules defined by the ISEL.

I will attach some print screens.

<img width="1210" height="834" alt="Image" src="https://github.com/user-attachments/assets/375f41ed-0331-4092-9509-2ca55f0b1613" />
<img width="1210" height="834" alt="Image" src="https://github.com/user-attachments/assets/71da214c-5493-444e-8c9d-351e7fe9256a" />

**Comment by daniel-m-carvalho** [comment]
The main issue for me  is that it takes to much time to compile everything. 

<img width="560" height="514" alt="Image" src="https://github.com/user-attachments/assets/bf2ad4fc-7d5c-4b03-9095-265ad4714484" />

**Comment by matpato** [comment]
Dear @daniel-m-carvalho 
Regarding this issue, there is nothing I can do on my side. Are you using a free plan? If so, my recommendation is to comment out the chapters you do not need, such as renaming the files (for example, changing chapter2.tex to chapter2_.tex). This will enable you to compile the document chapter you are currently using.
In the end, you will need to run the full compilation on your personal computer. I cannot resolve this problem, and it is not an issue with the template. All my students use my account, so I do not face this limitation. Best regards, MP
